mangaki/mangaki/utils/anidb.py

```diff
@@ -62,81 +55,30 @@ def get(self, id):
 
     r = self._request("anime", {'aid': id})
     soup = BeautifulSoup(r.text.encode('utf-8'), 'xml')  # http://stackoverflow.com/questions/31126831/beautifulsoup-with-xml-fails-to-parse-full-unicode-strings#comment50430922_31146912
-    """with open('backup.xml', 'w') as f:
-      f.write(r.text)"""
+    with open('backup.xml', 'w') as f:
+      f.write(r.text)
     if soup.error is not None:
       raise Exception(soup.error.string)
 
     anime = soup.anime
-    titles = anime.titles
-
-    a = Anime({
-      'id': id,
-      'type': str(anime.type.string),
-      'episodecount': int(anime.episodecount.string),
-      'startdate': datetime(*list(map(int, anime.startdate.string.split("-")))),
-      'enddate': datetime(*list(map(int, anime.enddate.string.split("-")))),
-      'titles': [(
-        str(title.string),
-        title['type'] if 'type' in title else "unknown"
-      ) for title in anime.find_all('title')],
-      'title': str(titles.find('title', attrs={'type': "main"}).string),
-      'relatedanime': [],
-      'url': str(anime.url.string) if anime.url else None,
-      'creators': anime.creators,
-      'description': str(anime.description.string),
-      'ratings': SmartDict({
-        'permanent': float(anime.ratings.permanent.string),
-        'temporary': float(anime.ratings.temporary.string),
-        'review': float(anime.ratings.review.string) if anime.ratings.review else ''
-      }),
-      'picture': "http://img7.anidb.net/pics/anime/" + str(anime.picture.string),
-      'categories': [],
-      'tags': [],
-      'characters': [],
-      'episodes': [],
-    })
-
-    self._cache[id] = a
-
+    all_titles = anime.titles
+    # creators = anime.creators
+    # episodes = anime.episodes
+    # tags = anime.tags
+    # characters = anime.characters
+    # ratings = anime.ratings.{permanent, temporary}
+
+    a = {
+      'title': str(all_titles.find('title', attrs={'type': "main"}).string),
+      'source': 'AniDB: ' + str(anime.url.string) if anime.url else None,
+      'ext_poster': 'http://img7.anidb.net/pics/anime/' + str(anime.picture.string),
+      # 'nsfw': ?
+      'date': datetime(*list(map(int, anime.startdate.string.split("-")))),
+      # not yet in model: 'enddate': datetime(*list(map(int, anime.enddate.string.split("-")))),
+      'synopsis': str(anime.description.string),
+      # 'artists': ? from anime.creators
+      'nb_episodes': int(anime.episodecount.string),
+      'anime_type': str(anime.type.string),
+      'anidb_aid': id
```

Comment:
It is generally a bad idea to name a variable `id`, time for riddles: why?

(answer: because it's supposed to be a built-in Python function.)